packages/stack/src/views/Stack/CardStack.tsx

```diff
@@ -420,7 +420,7 @@ export default class CardStack extends React.Component<Props, State> {
                     headerShown = isParentHeaderShown === false,
                   } = options;
 
-                  if (headerTransparent || headerShown === false) {
+                  if (!headerTransparent || headerShown === false) {
```

Comment:
The logic here is deciding whether to position the header using `position: 'absolute'` or `position: 'relative'`. The reason that we check `headerTransparent` is because for a transparent header, we want to layout the header over whatever is in the background. `position: 'relative'` means that the header will take up its own vertical space, which means it won't work well for the transparent-header scenario.